Enable `excludeEvent()` and `excludeHandler()`.

nouex opened the pull request:
Closes: #25

Commit:
lib, test: enable `excludeEvent()`

Commit:
lib, test: enable `excludeHandler()`

Commit:
test: fix fail

reecehudson commented:
TL;DR Travis-CI is a bitch.  One extraneous commit (within the perspective of pertaining to #25) is added just b/c.

I have just pushed the most recent commit (32f2882), because for some reason it passed the [Travis-CI](https://travis-ci.org/reecehudson/node-emitter-details/builds/139024384) builds (never running the new test files) but failed locally (not b/c of anything the previous commits do but I guess I just missed it previously.

reecehudson commented:
Still working on fixing the [CI](https://travis-ci.org/reecehudson/node-emitter-details/builds/139033638).  Since this fix does not pertain here, if it gets too deep and long I'll open a seperate PR for it.

Commit:
ci: fix build failing

Because of some unknown reason -- pretty sure it's cause the command `cd` is
a no-op on our Travis-CI build -- our `process.cwd()` in test/stack-trace.js
was not what is was expected to be (i.e. ./test) and so test/stack-trace.js
was causing the fail.

Now we simply skip that test *only if* we are on Travis-CI.

Refs: #27

Commit:
test: continuation of last commit

Commit:
test: continuation (2)

Commit:
test: continuation (3)

Commit:
test: continuation (3)

Commit:
test: continuation (4)

According to https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables

reecehudson commented:
Thank the lord Jesus Christ.  [Build passing](https://travis-ci.org/reecehudson/node-emitter-details).

reecehudson commented:
I know it got awful long but fuck it.

reecehudson commented:
...and since it got awful long why don't we add one more tiny change.

Commit:
ci: support node v4.4.4
